tests/house_tests.py

```diff
@@ -83,6 +83,24 @@ def test_filter_lowest_income(self):
         houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
         assert len(houses) == 1
 
+    def test_profession_either_parent(self):
+        HouseFactory(
+            country=Country.objects.get(name='Israel'),
+            city=City.objects.get(name='Tel Aviv'),
+            income=5000,
+            children=1,
+            parent_profession_2=Job.TEACHER,
+        ).save()
+        HouseFactory(
+            country=Country.objects.get(name='Israel'),
+            city=City.objects.get(name='Tel Aviv'),
+            income=5000,
+            children=1,
+            parent_profession_1=Job.TEACHER,
+        ).save()
+        houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
+        assert len(houses) == 2
```

Comment:
Can you add an assert which checks if the professions match the filter?

Reply:
ok I now added an assert that checks that Teacher is in profession 1 and 2 and in houses meaning was not filtered